test/spi/http/axiosHttpClientTest.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2018 Atomist, Inc.
+ *
+ * See LICENSE file.
+ */
+
+import * as http from "http";
+import "mocha";
+import * as os from "os";
+import * as assert from "power-assert";
+import { AxiosHttpClientFactory } from "../../../src/spi/http/axiosHttpClient";
+import { HttpMethod } from "../../../src/spi/http/httpClient";
+
+const hostname = os.hostname();
```

Comment:
I don't think this is reliable to bind against.

Reply:
I tried a couple of different things and all work locally on two different boxes but not on the build pods.